src/main/java/io/kokuwa/maven/helm/DependencyBuildMojo.java

```diff
@@ -37,6 +224,39 @@ public void execute() throws MojoExecutionException {
 		}
 
 		for (Path chartDirectory : getChartDirectories()) {
+			if (overwriteLocalDependencies) {
+				if (overwriteDependencyRepository == null || overwriteDependencyVersion == null) {
```

Comment:
A/Q: Can we default the version to something sensible? (Might be a future enhancement.)

Reply:
You could use the current version associated with the dependency if none is provided. Version is a required field so there will always be one for each dependency. So if the overwriteDependencyVersion is null, then the version will stay the same but the repo will get overwritten still.

Reply:
I think that makes sense